src/conversion.rs

```diff
@@ -51,6 +51,50 @@ where
     }
 }
 
+#[macro_use]
+macro_rules! try_from_wkt_impl {
+    ($($type: ident),+) => {
+        $(
+            /// Convert a Wkt enum into a specific geo-type
+            impl<T: CoordFloat> TryFrom<Wkt<T>> for geo_types::$type<T> {
+                type Error = Error;
+
+                fn try_from(mut wkt: Wkt<T>) -> Result<Self, Self::Error> {
+                    match wkt.items.len() {
+                        1 => {
+                            let item = wkt.items.pop().unwrap();
+                            let geometry = geo_types::Geometry::try_from(item)?;
+                            Self::try_from(geometry).map_err(|e| {
+                                match e {
+                                    geo_types::Error::MismatchedGeometry { expected, found } => {
+                                        Error::MismatchedGeometry { expected, found }
+                                    }
+                                    // currently only one error type in geo-types error enum, but that seems likely to change
+                                    #[allow(unreachable_patterns)]
+                                    other => Error::External(Box::new(other)),
```

Comment:
I thought it was worth future-proofing, so we don't have to bump WKT's minimum version of geo-types again if we add to the error enum.